internal/binder/function/funcs_agg.go

```diff
@@ -24,6 +25,203 @@
 	"github.com/lf-edge/ekuiper/pkg/cast"
 )
 
+var GlobalAggFuncs map[string]struct{}
+
+func init() {
+	GlobalAggFuncs = map[string]struct{}{}
+	GlobalAggFuncs["min"] = struct{}{}
+	GlobalAggFuncs["max"] = struct{}{}
+	GlobalAggFuncs["sum"] = struct{}{}
+	GlobalAggFuncs["avg"] = struct{}{}
+}
+
+func max(a, b float64) float64 {
+	if a > b {
+		return a
+	}
+	return b
+}
+
+func min(a, b float64) float64 {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func registerGlobalAggFunc() {
+	builtins["global_avg"] = builtinFunc{
```

Comment:
how about name it "cum_aum" which means cumulative sum?

Reply:
I think using `acc` (accumulate) as prefix is more clear.